src/core/sys/posix/time.d

```diff
@@ -126,8 +126,6 @@ else version (Solaris)
 else version (CRuntime_Bionic)
 {
     enum CLOCK_MONOTONIC        = 1;
-    enum CLOCK_MONOTONIC_RAW    = 4;
-    enum CLOCK_MONOTONIC_COARSE = 6;
```

Comment:
Better keep them, core.time use those, but nonstandard stuff should live here instead https://github.com/D-Programming-Language/druntime/blob/8d3f96a0cb5d08de7d352d0fdaa8221e46bc8aad/src/core/sys/linux/time.d.